CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Wazuh Chef v3.11.1_7.5.1
+
+### Added
+
+- Update to Wazuh version 3.11.1_7.5.1
+
```

Comment:
Please Add merged PR's to the Changelog:

- #78 
- #80 
- #74 

![image](https://user-images.githubusercontent.com/46649922/71903377-37bf5c80-3164-11ea-8e5c-50709489b825.png)

Reply:
Hi @jm404 !

CHANGELOG is now updated. https://github.com/wazuh/wazuh-chef/pull/82/commits/a580ca0d4876f57740a4b34aa9c0ae6106da5b41

Kr,

Rshad